packages/Frontend/src/components/Navigation/SidebarLeftOverlay.jsx

```diff
@@ -20,7 +20,7 @@ export default class SidebarLeftOverlay extends Component {
   }
 
   handleLogout = () => {
-    this.props.logout();
+    auth().signOut();
```

Comment:
We didn't need to pass down a logout prop from AppContainer any longer. If we just import firebase's auth into any file in order to interact with a user, in this case logging them out. You'll see later how logging a user out removes them from redux.